### DISCUSSION TITLE
Momentum Rotation for Demon Hunter

### DISCUSSION THREAD

**Discussion by xoGlobeXox** [Hekili/hekili#1263] [Q&A]
Hello, i am new on the website, i didn't know where to post this so i hope i didn't made a mistake.

I am using Hekili addon alot and i must say it's really a good ADDON, i play DH Demonic tallents and it's perfect.

I recently switched to Momentum, and i can see that the rotation that Hekili suggests its kinda wrong.

Is there something i need to adjust in the Hekili options for this or maybe something else.

Hope i will get an answer :)

Thank you

**Comment by Hekili** [comment]
Have you looked at your class settings?

/hekili > Havoc ?

**Reply by xoGlobeXox** [reply]
Hello i honestly don't know how to costumize the addon so Momentum rotation shows up.

If you can help me that would be great.

I apologize to distrurb.

Thank you very much

**Reply by Hekili** [reply]
Did you look where I said?

**Reply by xoGlobeXox** [reply]
Hello, yes i did ( Core, Toggles, Targeting, Performance ) can you tell me what to look for specifically ?

**Reply by Hekili** [reply]
Look at that first tab.  There are spec specific settings related to movement, no?

**Reply by xoGlobeXox** [reply]
Preferences 

( Recommend Movement )

Demon Blades

- warning --- if using Demon Blades .... etc ...


This ?

**Reply by Hekili** [reply]
Mmmhmm, look at Recommend Movement.
